app/assets/stylesheets/pageflow/editor/files_explorer.css.scss

```diff
@@ -33,6 +33,12 @@
 
   .entries_panel {
     width: 20%;
+
+    li.blank_slate {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `blank_slate` should be written in lowercase with hyphens